packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec

```diff
@@ -6,8 +6,8 @@
 %global plugin_name discovery
 %global foreman_min_version 1.24.0
```

Comment:
Is this still correct?

Reply:
Technically yes, I haven't tried tho. But I don't see any incompatibility.